packages/edit-post/src/components/sidebar/style.scss

```diff
@@ -25,7 +26,7 @@
 		position: relative;
 
 		@include break-small() {
-			overflow: hidden;
+			overflow: visible;
 			height: auto;
 			max-height: none;
 		}
```

Comment:
Not sure I understand why the changes in this file are needed for this particular PR?

Reply:
https://github.com/WordPress/gutenberg/pull/17926#pullrequestreview-312188131

Reply:
Why it doesn't just use Popover, this issue is already solved there, and I think the changes here might cause more unexpected bugs (like the scrolling issue we noticed)

Reply:
Is the `Popover` fully accessible?

cc @tellthemachines @enriquesanchez

Reply:
If it's not it should be :) but we did a lot of a11y work on Popover back then.

Reply:
I'm not aware of any accessibility issues with `Popover`.

Reply:
There are no issues that I know of with `Popover` itself, but we had some trouble with focus management when using it in our previous attempt to implement this component: https://github.com/WordPress/gutenberg/pull/17418#pullrequestreview-294158647

Reply:
Yes, that's what I was thinking of.

Part of the reason I didn't use `Popover` here from the start is that previous attempts had failed with it.